util/io/src/worker.rs

```diff
@@ -68,11 +66,8 @@ impl Worker {
 		};
 		worker.thread = Some(thread::Builder::new().name(format!("IO Worker #{}", index)).spawn(
 			move || {
-				panic_handler.catch_panic(move || {
-					Worker::work_loop(stealer, channel.clone(), wait, wait_mutex.clone(), deleting)
-				}).unwrap()
-			})
-			.expect("Error creating worker thread"));
+				Worker::work_loop(stealer, channel.clone(), wait, wait_mutex.clone(), deleting)
```

Comment:
That's fine. rocksdb and Parity are designed to be crash resistant. Removing unwinding info speeds up compilation and reduces binary size. Cargo support for this feature seems to be buggy though. Will have to revert to panic handler for now